### PR TITLE
Add footer #19

### LINKED ISSUE #19: Add footer

### PR DESCRIPTION
**Pull request by julsfelic**
Added the footer with social icons to the site. The twitter and facebook icons don't point to anything at 
the moment but the github redirects to this repo.

Closes #19 

@scottfirestone @Salvi6God @Draganovic

### CODE REVIEW

File: app/assets/images/facebook.svg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (ghost) [inline app/assets/images/facebook.svg:39]: Is there are reason why there are so many "g" elements?

**Comment by ghost** [comment]
Other than the comment I made, everything seems good.

File: app/assets/images/facebook.svg (hunk 1 above)
Comment (scottfirestone) [inline app/assets/images/facebook.svg:39]: I have the same question.

**Comment by scottfirestone** [comment]
Curious to hear about the <g> tags. Also we have merge conflicts, would like to walk through it with you at least for this first one. We'll regroup later today on this @julsfelic

File: app/assets/images/facebook.svg (hunk 1 above)
Comment (julsfelic) [inline app/assets/images/facebook.svg:39]: It is like if you were to look at an image file in you text editor it shows a bunch of random characters. SVG is the same way.

**Comment by julsfelic** [comment]
Commented on why the SVG has all those tags and fixed the merge conflict. @scottfirestone @Salvi6God @Draganovic

**Comment by scottfirestone** [comment]
Thanks for explanation and fixing merge conflicts. Merging now. @julsfelic

**Pull request merged**